update/Pos/Update/Core/Types.hs

```diff
@@ -58,16 +58,16 @@ import           Serokell.Util.Text         (listJson)
 import           Universum                  hiding (show)
 
 import           Pos.Binary.Class           (Bi, Raw)
-import           Pos.Binary.Crypto          ()
+import           Pos.Core.Address           (addressHash)
+import           Pos.Core.Coin              ()
+import           Pos.Core.Script            ()
+import           Pos.Core.Types             (BlockVersion, CoinPortion, FlatSlotId,
+                                             ScriptVersion, SoftwareVersion)
+import           Pos.Core.Types             ()
+import           Pos.Core.Version           ()
```

Comment:
I think importing 6 modules instead of `Pos.Core` is fine if you want it to be easier to find these names, but here 4 imports are empty, so I would import `Pos.Core ()` instead of 4 such imports.